src/common/images/imageGallery.component.tsx

```diff
@@ -0,0 +1,435 @@
+        paddingLeft: '8px',
+      }),
+
+    renderRowActionMenuItems: ({ closeMenu, row }) => {
+      return [
+        <MenuItem
+          key="edit"
+          aria-label={`Edit ${row.original.file_name} image`}
+          sx={{ m: 0 }}
+        >
+          <ListItemIcon>
+            <EditIcon />
+          </ListItemIcon>
+          <ListItemText>Edit</ListItemText>
+        </MenuItem>,
+        <MenuItem
+          key="download"
+          aria-label={`Download ${row.original.file_name} image`}
+          sx={{ m: 0 }}
+        >
+          <ListItemIcon>
+            <DownloadIcon />
+          </ListItemIcon>
+          <ListItemText>Download</ListItemText>
+        </MenuItem>,
+        <MenuItem
+          key="info"
+          aria-label={`Show ${row.original.file_name} image information`}
+          onClick={() => {
+            setSelectedImage(row.original);
+            setOpenMenuDialog('information');
+            closeMenu();
+          }}
+          sx={{ m: 0 }}
+        >
+          <ListItemIcon>
+            <InfoOutlinedIcon />
+          </ListItemIcon>
+          <ListItemText>Information</ListItemText>
+        </MenuItem>,
+        <MenuItem
+          key="delete"
+          aria-label={`Delete catalogue item ${row.original.file_name}`}
+          sx={{ m: 0 }}
+        >
+          <ListItemIcon>
+            <DeleteIcon />
+          </ListItemIcon>
+          <ListItemText>Delete</ListItemText>
+        </MenuItem>,
+      ];
+    },
+  });
+
+  const [isCollapsed, setIsCollapsed] = React.useState(true);
+
+  const handleToggle = () => {
+    setIsCollapsed(!isCollapsed);
+  };
+  const data = table
+    .getSortedRowModel()
+    .rows.map((row) => row.getVisibleCells().map((cell) => cell)[0]);
+  const displayedImages = table
+    .getPaginationRowModel()
+    .rows.map(
+      (row) => row.getVisibleCells().map((cell) => cell.row.original)[0]
+    );
+
+  return (
+    <>
+      {!imageIsLoading ? (
+        (!images || images.length === 0) && (
+          <Box
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+            height="300px"
+            flexDirection="column"
+            textAlign="center"
+          >
+            <Typography variant="h6" fontWeight="bold">
+              No images available
+            </Typography>
+            <Typography variant="body1">
+              Please add an image by opening the Action Menu and clicking the
+              Upload Images menu item.
+            </Typography>
+          </Box>
+        )
+      ) : (
+        <Box sx={{ width: '100%' }}>
+          <LinearProgress />
+        </Box>
+      )}
+
+      {images && images.length !== 0 && (
+        <Grid container>
+          <Grid item container mt={2} direction="column" alignItems="center">
+            <Collapse in={!isCollapsed} style={{ width: '100%' }}>
+              <Grid marginTop={'auto'} direction="row" item container>
```

Comment:
```suggestion
              <Grid item container marginTop={'auto'} direction="row">
```
Didn't see it originally as we normally have them first, there is another below. (I still think component makes sense before these though)